open_discussions/views_test.py

```diff
@@ -19,11 +19,8 @@
 @pytest.mark.parametrize(
     "test_user,expect_auth", [[lazy("logged_in_user"), True], [None, False]]
 )
-def test_webpack_url(settings, mocker, client, test_user, expect_auth):
+def test_webpack_url(settings, client, test_user, expect_auth):
     """Verify that webpack bundle src shows up in production"""
-    get_bundle_mock = mocker.patch(
-        "open_discussions.templatetags.render_bundle._get_bundle"
-    )
```

Comment:
I'm still trying to figure out how to spy on webpack_loader's `render_bundle` now that we are not using our custom one.

Reply:
IMHO I would just remove this test.